.gitignore

```diff
@@ -46,3 +46,5 @@ _build/
 
 # Tests and docs
 .coverage
+
+!Flattening development/images/*.png
```

Comment:
Useful to have this ignored locally whilst developing but `.gitignore` should be general and work for everyone and most won't have this.

Simply not staging the file would be fine in future when you add folders during develpoment.

Reply:
I think that this was removed already as of [2737cf8](https://github.com/AFM-SPM/TopoStats/pull/351/commits/2737cf8c9741ddade9651b3f2f9f9d264ec00e60). Sorry if I've got this wrong.